### PR TITLE
Em 3379 fix field validation

### PR DESCRIPTION
**Pull request by yogesh-hullatti**
https://tools.hmcts.net/jira/browse/EM-3379

### CODE REVIEW

File: src/aat/java/uk/gov/hmcts/reform/em/orchestrator/functional/AutomatedBundlingScenarios.java [hunk 1]
```diff
@@ -55,6 +58,15 @@ public void testInvalidConfig() {
                 response.getBody().jsonPath().getString("errors[0]"));
     }
 
+    @Test
+    public void test51CharsFileName() {
+        Response response = postNewBundle(filenameWith51CharsJson);
+
+        assertEquals(200, response.getStatusCode());
```
Comment (benouaer) [inline src/aat/java/uk/gov/hmcts/reform/em/orchestrator/functional/AutomatedBundlingScenarios.java:65]: why do we return 200 if there's an error?

File: src/test/java/uk/gov/hmcts/reform/em/orchestrator/automatedbundling/AutomatedCaseUpdaterTest.java [hunk 2]
```diff
@@ -123,50 +125,69 @@ public void createCaseBundlesPropertyWhenItDoesntExist() throws IOException {
...
-                                        + "\"stitchStatus\": null,\n"
-                                        + "\"paginationStyle\": \"topLeft\",\n"
-                                        + "\"pageNumberFormat\": \"numberOfPages\",\n"
-                                        + "\"stitchingFailureMessage\": null\n"
-                                        + "}\n"
-                                        + "}]}}}")
-                        ));
+            .thenReturn(
+                new BufferedReader(
+                    new StringReader("{\"case_details\":{\"case_data\": {\"bundleConfiguration\":\"example.yaml\","
```
Comment (benouaer) [inline src/test/java/uk/gov/hmcts/reform/em/orchestrator/automatedbundling/AutomatedCaseUpdaterTest.java:154]: why don't we just use a test bundle file?

File: src/aat/java/uk/gov/hmcts/reform/em/orchestrator/functional/AutomatedBundlingScenarios.java (hunk 1 above)
Comment (yogesh-hullatti) [reply src/aat/java/uk/gov/hmcts/reform/em/orchestrator/functional/AutomatedBundlingScenarios.java:65]: This is current implementation where in the consuming service is expected to look at the errors before looking in to the body of the response.

File: src/test/java/uk/gov/hmcts/reform/em/orchestrator/automatedbundling/AutomatedCaseUpdaterTest.java (hunk 2 above)
Comment (yogesh-hullatti) [reply src/test/java/uk/gov/hmcts/reform/em/orchestrator/automatedbundling/AutomatedCaseUpdaterTest.java:154]: This is case data.

File: src/aat/java/uk/gov/hmcts/reform/em/orchestrator/functional/AutomatedBundlingScenarios.java (hunk 1 above)
Comment (yogesh-hullatti) [reply src/aat/java/uk/gov/hmcts/reform/em/orchestrator/functional/AutomatedBundlingScenarios.java:65]: I will create separate JIRA ticket to chase this with the consuming services in terms of returning Error response code.

**Review by benouaer — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/em/orchestrator/config/Constants.java [hunk 3]
```diff
@@ -5,11 +5,11 @@
  */
 public final class Constants {
 
-    public static final String SYSTEM_ACCOUNT = "system";
-    public static final String ANONYMOUS_USER = "anonymoususer";
-    public static final String DEFAULT_LANGUAGE = "en";
     public static final String STITCHED_FILE_NAME_FIELD_LENGTH_ERROR_MSG = "File Name should contain at least 2 and "
         + "not more than 50 Chars";
+    public static final String FILE_NAME_WITH_51_CHARS_LENGTH = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
```
Comment (benouaer) [inline src/main/java/uk/gov/hmcts/reform/em/orchestrator/config/Constants.java:10]: seems like we're mixing test code with main code here

**Comment by yogesh-hullatti** [comment]
Not required

**Pull request closed without merging**